doc/role_whitelist_management.md

```diff
@@ -153,6 +174,7 @@ Four types of resources can be managed in whitelists:
 * data - represent all kinds of data resources in Cassandra such as data in tables
 * functions - represent all kinds of function and aggregate resources in Cassandra
 * roles - represent all kinds of role resources in Cassandra
+* grants - represent a grant on a wrapped resource (with one of the above types)
```

Comment:
Change text above to:
Five types of resources...

Reply:
done